Makefile

```diff
@@ -96,90 +102,41 @@ lint: isort-check flake8
 ## Collect the messages to translate
 .PHONY: gettext-collect
 gettext-collect:
-	pushd ./creme/products && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		popd;
-	pushd ./creme/opportunities && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		popd;
-	pushd ./creme/mobile && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i tests.py && \
-		popd;
-	pushd ./creme/reports && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		django-admin.py makemessages -d djangojs -l ${CREME_LANGUAGE} -i "static/reports/js/tests/*" && \
-		popd;
-	pushd ./creme/creme_core && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" -i "templates/creme_core/tests/*" && \
-		django-admin.py makemessages -d djangojs -l ${CREME_LANGUAGE} -i "static/creme_core/js/tests/*" && \
-		popd;
-	pushd ./creme/sms && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		popd;
-	pushd ./creme/events && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i tests.py && \
-		popd;
-	pushd ./creme/projects && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i tests.py && \
-		django-admin.py makemessages -d djangojs -l ${CREME_LANGUAGE} -i "static/projects/js/tests/*" && \
-		popd;
-	pushd ./creme/crudity && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" -e py -e html -e xml && \
-		popd;
-	pushd ./creme/emails && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		django-admin.py makemessages -d djangojs -l ${CREME_LANGUAGE} -i "static/emails/js/tests/*" && \
-		popd;
-	pushd ./creme/tickets && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i tests.py && \
-		popd;
-	pushd ./creme/commercial && \
-		django-admin.py makemessages -l ${CREME_LANGUAGE} -i "tests/*" && \
-		popd;
-	pushd ./creme && \
```

Comment:
This one is missing in the new target

Reply:
https://github.com/HybirdCorp/creme_crm/blob/master/creme/locale/fr/LC_MESSAGES/django.po#L5

Reply:
Good point. Done by an ugly hack but it will be cleaned along usage of new localisation tools for the resolution of issue #22